x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/planner/QueryTranslator.java

```diff
@@ -459,10 +459,10 @@ protected QueryTranslation asQuery(MultiMatchQueryPredicate q, boolean onAggs) {
         }
     }
 
-    static class BinaryLogic extends ExpressionTranslator<BinaryPredicate> {
+    static class BinaryLogic extends ExpressionTranslator<org.elasticsearch.xpack.sql.expression.predicate.logical.BinaryLogic> {
 
         @Override
-        protected QueryTranslation asQuery(BinaryPredicate e, boolean onAggs) {
+        protected QueryTranslation asQuery(org.elasticsearch.xpack.sql.expression.predicate.logical.BinaryLogic e, boolean onAggs) {
```

Comment:
and here?

Reply:
There's a `BinaryLogic` class declared inside `QueryTranslator` (the one above).